core/src/test/java/org/apache/iceberg/TestTableMetadata.java

```diff
@@ -1685,4 +1685,82 @@ public void testV3TimestampNanoTypeSupport() {
         ImmutableMap.of(),
         3);
   }
+
+  @Test
+  public void updatingMetadataLocationViaBuilderUpdatesTimestampAndWritesMetadataLogEntry() {
```

Comment:
Is there a more concise way of saying this? These method names are longer than people will actually read.

Reply:
yeah sorry, I had a hard time finding a method name that reflects what's being done. I've shortened it